### PR TITLE
update clause 7 for editorial changes. 

### PR DESCRIPTION
**Pull request by amilan17**
Also see questions or suggestions inline.

### CODE REVIEW

File: standard/requirements/core/REQ_releasing.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/releasing*
-^|A |Final approval of the WTH updates will go through the WMO fast-track amendment process.footnote:[https://community.wmo.int/en/activity-areas/wis/amendment-processes-wis-manuals-and-guides]
-^|B |Addition of a new centre identifier SHALL trigger an **immediate** stable release of WTH updates, which is not required to align with the WMO fast-track amendment process.footnote:[https://community.wmo.int/en/activity-areas/wis/amendment-processes-wis-manuals-and-guides]
-^|C |Immediate stable releases SHALL only contain changes resulting from a new centre identifier.
+^|A |Updates to the discipline topics (level 8 and beyond) will go through the WMO fast-track approval procedure.footnote:[https://community.wmo.int/en/activity-areas/wis/amendment-processes-wis-manuals-and-guides]
```
Comment (amilan17) [inline standard/requirements/core/REQ_releasing.adoc:5]: add a requirement: 
"Updates for major versions will go through the WMO standard approval procedure..."

File: standard/recommendations/core/REC_publishing.adoc [hunk 2]
```diff
@@ -2,5 +2,5 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Recommendation {counter:req-id}* |*/rec/core/publishing*
-^|A |The domain ``experimental`` level SHOULD be used as a temporary approach until a given domain topic is approved.
+^|A |The topic level ``experimental`` should be used as a temporary approach until a given sub-discipline topic is approved.
```
Comment (amilan17) [inline standard/recommendations/core/REC_publishing.adoc:5]: normalize "sub-discipline topic"?

File: standard/requirements/core/REQ_management.adoc [hunk 3]
```diff
@@ -2,18 +2,8 @@
...
-
-- domain specific teams submit proposals to TT-WISMD
-
-- TT-WISMD performs a review of the proposal and curates the content for consistency.  Discussion and clarifications on the proposal are performed as required
-
-- once consensus is reached, TT-WISMD approves the proposal for inclusion into the next release of WTH
-
-- Final approval of the WTH updates will go through the WMO fast-track amendment process.footnote:[https://community.wmo.int/en/activity-areas/wis/amendment-processes-wis-manuals-and-guides]
+^|A |Primary levels 1-7 shall be determined by WMO.
+^|B |Discipline specific levels (level 8 and beyond) shall be proposed by domain experts and user communities.
```
Comment (amilan17) [inline standard/requirements/core/REQ_management.adoc:6]: normalize "discipline specific levels (level 8 and beyond)"?

File: standard/sections/clause_7_normative_text.adoc [hunk 4]
```diff
@@ -28,62 +32,63 @@ The table below provides an overview primary topic levels of the WIS2 topic hier
...
 |===
 
 
 ==== Management
+{{_I think these two paragraphs belong in the introduction_}}
 
-The WTH is managed by the WMO. Updates to the WTH are made as part of two workflows: core levels, and domain specific levels.
-
-The WIS Topic Hierarchy has been developed according to the classification of the Earth System domains in Annex 1 of Resolution 1 Cg-Ext(2021), and it is managed in two parts:
+The WTH is managed by the WMO. Updates to the WTH are made as part of two workflows: primary levels and domain specific levels.
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:59]: normalize "domain specific levels"?

File: standard/sections/clause_7_normative_text.adoc [hunk 5]
```diff
@@ -28,62 +32,63 @@ The table below provides an overview primary topic levels of the WIS2 topic hier
...
 
 1. Primary topic levels (levels 1-7): topic structure applies to all data and services in WIS.  They are relational, meaning that any combination of each level can be used to construct a topic applicable to a data or metadata notification
-2. Domain specific topic subcategory levels (level 8 and beyond): topic structures proposed by domain experts and user communities.  Note that the number of levels in this part may vary according to the requirements of various domains, and are meant to represent a hierarchical representation of data publication from a given domain.
+2. Domain specific topic subcategory levels (level 8 and beyond): topic structures proposed by domain experts and user communities.  These levels are a hierarchical representation of the dataset and the the number of levels in this part may vary according to the requirements of various domains.
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:62]: normalize "Domain specific topic subcategory levels (level 8 and beyond)"

**Review by tomkralidis — COMMENTED** [review]
Per comments.  Great work @amilan17

File: standard/requirements/core/REQ_releasing.adoc [hunk 6]
```diff
@@ -2,7 +2,9 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/releasing*
-^|A |Final approval of the WTH updates will go through the WMO fast-track amendment process.footnote:[https://community.wmo.int/en/activity-areas/wis/amendment-processes-wis-manuals-and-guides]
-^|B |Addition of a new centre identifier SHALL trigger an **immediate** stable release of WTH updates, which is not required to align with the WMO fast-track amendment process.footnote:[https://community.wmo.int/en/activity-areas/wis/amendment-processes-wis-manuals-and-guides]
-^|C |Immediate stable releases SHALL only contain changes resulting from a new centre identifier.
+//re-ordered the reqs
```
Comment (amilan17) [inline standard/requirements/core/REQ_releasing.adoc:5]: is this order ok?

File: standard/sections/clause_7_normative_text.adoc [hunk 7]
```diff
@@ -1,18 +1,26 @@
...
 
-This Core Conformance Class provides requirements to articulate the required elements of the definition and management of the WIS2 Topic Hierarchy.
+This Core Conformance Class provides requirements for the definition and management of the WIS2 Topic Hierarchy.
 
 include::../requirements/requirements_class_core.adoc[]
 
-The standard topic structure ensures that the WIS2 ecosystem (data publisher, data user, and global services) is a robust, effective, and unified data exchange platform for weather, climate, and water
+The WTH is composed of primary topics (levels 1-7) and sub-discipline specific topics (level 8 and beyond).
+//change .."and services" to "..and resources"?
+The primary topics apply to all data and services in WIS.  They are relational, meaning that any combination of the values in each level can be used to construct a topic applicable to a notification.
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:17]:
```suggestion
The primary topics apply to all data, metadata, and reports in WIS. They are relational, meaning that any combination of the values in each level can be used to construct a topic applicable to a notification.
```

File: standard/sections/clause_7_normative_text.adoc [hunk 8]
```diff
@@ -24,39 +32,67 @@ The table below provides an overview primary topic levels of the WIS2 topic hier
...
 
-The WIS Topic Hierarchy has been developed according to the classification of the Earth System domains in Annex 1 of Resolution 1 Cg-Ext(2021), and it is managed in two parts:
+include::../requirements/core/REQ_publishing.adoc[]
+include::../recommendations/core/REC_publishing.adoc[]
+include::../recommendations/core/PER_publishing.adoc[]
 
-1. Primary topic levels (levels 1-7): topic structure applies to all data and services in WIS.  They are relational, meaning that any combination of each level can be used to construct a topic applicable to a data or metadata notification
-2. Domain specific topic subcategory levels (level 8 and beyond): topic structures proposed by domain experts and user communities.  Note that the number of levels in this part may vary according to the requirements of various domains, and are meant to represent a hierarchical representation of data publication from a given domain.
+==== Management
+//new text
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:94]: new text below

File: standard/requirements/core/REQ_versioning.adoc [hunk 9]
```diff
@@ -7,7 +7,8 @@
 ^|C |A change in structure of the topic hierarchy SHALL result in a major version update.
 ^|D |A new topic SHALL NOT result in any version update.
 ^|E |A new centre identifier SHALL NOT result in any version update.
-^|F |A major version SHALL result in a change / bump to the version level (i.e. ``a`` -> ``b``).
+^|F |A major version SHALL result in a change to the version level (i.e. ``a`` -> ``b``).
 ^|G |A minor version SHALL NOT result in any changes to the version level.
-^|H |A renaming or removal in the WMO Notification Message encoding SHALL result in a major version update.
+^|H |A renaming or removal in the WMO Notification Message encoding SHALL result in a major version update.
```
Comment (amilan17) [inline standard/requirements/core/REQ_versioning.adoc:12]: Is this correct? This rule seems to be for WNM, NOT WTH...

**Review by amilan17 — COMMENTED** [review]
some questions to review inline

File: standard/requirements/core/REQ_versioning.adoc [hunk 10]
```diff
@@ -7,7 +7,8 @@
 ^|C |A change in structure of the topic hierarchy SHALL result in a major version update.
 ^|D |A new topic SHALL NOT result in any version update.
 ^|E |A new centre identifier SHALL NOT result in any version update.
-^|F |A major version SHALL result in a change / bump to the version level (i.e. ``a`` -> ``b``).
+^|F |A major version SHALL result in a change to the version level (i.e. ``a`` -> ``b``).
 ^|G |A minor version SHALL NOT result in any changes to the version level.
-^|H |A renaming or removal in the WMO Notification Message encoding SHALL result in a major version update.
+^|H |A renaming or removal in the WMO Notification Message encoding SHALL result in a major version update. 
+// Is req. H correct? When a WNM changes versions then the WTH shall have a major version update too?
```
Comment (tomkralidis) [inline standard/requirements/core/REQ_versioning.adoc:13]: Yes, req. H is correct/intended.

File: standard/sections/clause_7_normative_text.adoc [hunk 11]
```diff
@@ -1,18 +1,26 @@
 == The WIS2 Topic Hierarchy
 
-The WIS Topic Hierarchy (WTH) provides a structure used by data providers and WIS Global Services in support of core WIS workflows: publish, discover, subscribe and download.
+The WIS Topic Hierarchy (WTH) provides a structure used by data providers and WIS Global Services in support of core WIS workflows: publish, discover, subscribe and download. 
+
+The WIS2 Topic Hierarchy is included in the discovery metadata record <<see section X>>, made available by WIS brokers <<see Y>>, and selected by users to subscribe and receive notifications <<see Z>>.
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:5]: The WIS2 Topic Hierarchy is provides a mechanism for users to subscribe to and receive data or metadata notifications, and is used in discovery metadata records as well as WIS2 brokers.

File: standard/sections/clause_7_normative_text.adoc [hunk 12]
```diff
@@ -1,18 +1,26 @@
...
 ==== Overview
 
-This Core Conformance Class provides requirements to articulate the required elements of the definition and management of the WIS2 Topic Hierarchy.
+This Core Conformance Class provides requirements for the definition and management of the WIS2 Topic Hierarchy.
 
 include::../requirements/requirements_class_core.adoc[]
 
-The standard topic structure ensures that the WIS2 ecosystem (data publisher, data user, and global services) is a robust, effective, and unified data exchange platform for weather, climate, and water
+The WTH is composed of primary topics (levels 1-7) and sub-discipline specific topics (level 8 and beyond).
+//change .."and services" to "..and resources"?
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:16]: +1 to change "and services" to "and resources", please update?

File: standard/sections/clause_7_normative_text.adoc [hunk 13]
```diff
@@ -1,18 +1,26 @@
...
+The primary topics apply to all data and services in WIS.  They are relational, meaning that any combination of the values in each level can be used to construct a topic applicable to a notification.
+
+The sub-discipline topics are proposed by domain experts and user communities.  These levels are a hierarchical representation of the dataset and the number of levels in this part may vary according to the requirements of various domains.
 
-The table below provides an overview primary topic levels of the WIS2 topic hierarchy.
+The representation is encoded as a simple text string of values in each topic level separated by a "/". For example, ``origin/a/wis2/data/ca-eccc/core/weather/surface-based-observations/synop`` or ``origin/a/wis2/data/ca-eccc/recommended/atmospheric-composition/experimental/space-based-observation/geostationary/solar-flares``.
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:21]: Change "/" to ``/``

File: standard/sections/clause_7_normative_text.adoc [hunk 14]
```diff
@@ -24,39 +32,67 @@ The table below provides an overview primary topic levels of the WIS2 topic hier
...
+|Data policy as defined by the WMO Unified Data Policy. ``core`` data are available with open access on a free and unrestricted basis and ``recommended`` data are available from the original NC/DCPC which may require authentication or authorization
 
 |7
 |earth-system-discipline
-|As per Annex 1 of resolution 1 Cg-Ext-2021
+|Seven high-level categories as defined by the WMO Unified Data Policy, Annex 1: (``atmospheric-composition``, ``climate``, ``cryosphere``, ``hydrology``, ``ocean``, ``space-weather``, or ``weather``)
 
 |===
 
+.WTH example table of sub-discipline topics
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:59]: Suggest we introduce this table at later time, when WTH is populated further.

File: standard/sections/clause_7_normative_text.adoc [hunk 15]
```diff
@@ -24,39 +32,67 @@ The table below provides an overview primary topic levels of the WIS2 topic hier
...
+|
+|
+|temp
+|===
 
-==== Management
+==== Publishing guidelines
 
-The WTH is managed by the WMO. Updates to the WTH are made as part of two workflows: core levels, and domain specific levels.
+For maximum utility and efficient management of topics, it is recommended that ``data``, ``metadata``, and ``reports`` need to be published to a detailed level. This helps with avoiding "pollution" of messages under the primary topics.  Note that each discipline has a sub-discipline topic named ``experimental`` for publication to provisional topics.
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:87]: Remove "reports".

File: standard/sections/clause_7_normative_text.adoc [hunk 16]
```diff
@@ -75,16 +111,19 @@ include::../requirements/core/REQ_conventions.adoc[]
...
+=== All topics
+A zip file of all the approved topics is available at https://schemas.wmo.int/wis2/wth/a/1.0/wth.zip.
 
-include::../requirements/core/REQ_publishing.adoc[]
-include::../recommendations/core/REC_publishing.adoc[]
-include::../recommendations/core/PER_publishing.adoc[]
+A browseable and searchable codelist of all topics can be found at https://codes.wmo.int/wis2/wth.
+
+=== Compliance
+Conformance with this standard shall be checked using the tests specified the Conformance Class Abstract Test Suite (Normative). It is published at https://schemas.wmo.int/wis2/wth/ats.
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:129]: https://schemas.wmo.int/wth/a/ats

File: standard/sections/clause_7_normative_text.adoc [hunk 17]
```diff
@@ -75,16 +111,19 @@ include::../requirements/core/REQ_conventions.adoc[]
...
 include::../recommendations/core/REC_centre-id.adoc[]
 include::../recommendations/core/PER_centre-id.adoc[]
 
-==== Publishing guidelines
+== Resources
+//I don't think we need more examples for the WTH
 
-For maximum utility and efficient management of topics, data, metadata, and reports need to be published to a minimum level.  This helps with avoiding "pollution" of messages in higher level topics which are used to delineate core concepts (centre identification, notification types, data policy, etc.).  Note that each domain is designated a topic named ``experimental`` for publication to provisional topics.
+=== All topics
+A zip file of all the approved topics is available at https://schemas.wmo.int/wis2/wth/a/1.0/wth.zip.
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:124]: https://schemas.wmo.int/wth/a/wth-a.zip

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: standard/sections/clause_7_normative_text.adoc [hunk 18]
```diff
@@ -24,39 +30,67 @@ The table below provides an overview primary topic levels of the WIS2 topic hier
...
+|Fixed value of ``wis2`` for WIS2
 
 |4
 |centre-id
 |Acronym as specified by member and endorsed by the PR of the country and WMO
 
 |5
 |notification-type
-|WIS2 notification types (data, metadata, reports [from monitoring activities])
+|WIS2 notification types (``data``, ``metadata``, or ``reports`` [from monitoring activities])
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:45]: Do we remove reports from here too?

File: standard/requirements/core/REQ_versioning.adoc (hunk 10 above)
Comment (amilan17) [reply standard/requirements/core/REQ_versioning.adoc]: Ok. I don't understand what it means....

File: standard/sections/clause_7_normative_text.adoc (hunk 14 above)
Comment (amilan17) [reply standard/sections/clause_7_normative_text.adoc:57]: perhaps. @wmo-im/tt-wismd -- what do you think of this table?

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**